receiver/tlscheckreceiver/metadata.yaml

```diff
@@ -0,0 +1,35 @@
+type: tlscheck
+
+status:
+  class: receiver
+  stability:
+    development: [metrics]
+  distributions: [contrib]
+  codeowners:
+    active: [michael-burt]
```

Comment:
Please also include @atoulme as a code owner since they sponsored this component.

Reply:
fixed